twisted/plugins/multichain_crawler.service

```diff
@@ -0,0 +1,14 @@
+[Unit]
```

Comment:
Unrelated to this PR but maybe we should also create `.service` files for the other twistd plugins?